### PR TITLE
update broken links in application requirements

### PR DESCRIPTION
**Pull request by varodrig**
- CR, per K8s docs, was pointing to an incorrect path.
- Graduation criteria were pointing to an existing document

### CODE REVIEW

**Comment by varodrig** [comment]
@hbelmiro to review

File: guidelines/application_requirements.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ This document is not intended to cover SDKs (e.g. fairing) or client side tools
 ## Reference
 
 * TFJob and PyTorch 1.0 [exit criterion](http://bit.ly/operators-exit-criterion)
-* [CNCF Graduation Criteria](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc)
+* [CNCF Graduation Criteria](https://github.com/cncf/toc/blob/main/process/graduation_criteria.md)
```
Comment (hbelmiro) [inline guidelines/application_requirements.md:96]: Shouldn't we point to https://github.com/cncf/toc/blob/main/.github/ISSUE_TEMPLATE/template-graduation-application.md instead?

Comment (varodrig) [reply guidelines/application_requirements.md]: yes, you are right. udpating the PR

**Review by hbelmiro — COMMENTED** [review]
/lgtm

**Review by andreyvelich — COMMENTED** [review]
Thanks for the update @varodrig!
/lgtm
/approve

**Pull request merged**